### ISSUE TITLE
Issue with tests for step 66 of platformer game project

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#54113]
### Describe the Issue

We had a camper on the [forum](https://forum.freecodecamp.org/t/learn-intermediate-oop-by-building-a-platformer-game-step-66/679792) unable to pass the test with the following answer

```js
this.position = {
      x: x,
      y: y,
    };
```
the test is expecting the shorthand notation
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions

```js
this.position = {
      x,
      y,
    };
```
this could either be a good opportunity to teach that

or 

update the tests to accept either valid answer

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-intermediate-oop-by-building-a-platformer-game/step-66

### Your code

see explanation above

### Expected behavior

see explanation above

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by lasjorg** [comment]
I think we should update the requirements and make it clear it must use the shorthand syntax.

If the shorthand syntax hasn't been taught yet (which I would assume it has if it is the requirement) we should teach it.

**Comment by jdwilkin4** [comment]
Yeah, I just looked through the previous project and it hasn't been taught yet.

So I just updated the issue for the suggested new syntax

**Comment by natikaggarwal** [comment]
I would love to work on this issue, can you please assign this to me?

**Comment by jdwilkin4** [comment]
@natikaggarwal 

We don't assign issues except to longtime contributors. 
I just opened this up for first timers only

**Comment by jdwilkin4** [comment]
# Note to contributors
This has been opened for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://discord.com/channels/692816967895220344/715074489422970962), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

## Files that need to be changed
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64c74a8a4138c6032241d498.md

Happy coding

**Comment by natikaggarwal** [comment]
@jdwilkin4 Thank you for the reply, when this issue says update directions, does it mean to update "description" section or the "hints" section?

**Comment by jdwilkin4** [comment]
@natikaggarwal 

update the description and the second hint to mention the use of the shorthand syntax
